packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -530,8 +616,22 @@ export function TransferPanel() {
         } else {
           const amountRaw = utils.parseUnits(amount, 18)
 
+          if (isSmartContractWallet) {
+            showDelayedSCTxRequest()
```

Comment:
UX suggestion: When transferring from Goerli to Arbgoerli using safe, the assets did not reflect in the ArbGoerli wallet till 6 minutes.. till then I was figuring out what went wrong with the transactions. Maybe we can highlight the same in our UI? This will give the user much more visual indication compared to just a tooltip coming on the button.

Example mockup- 
<img width="1059" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/7558499/e459b082-b60a-40e0-8086-5de7c513bd79">

Reply:
that makes sense though i'm no longer sure if it should be handled in this PR
it seems like we need to do a thorough UX review for the whole custom destination address flow and that will possibly surface many more UX issues